wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -254,7 +254,13 @@ data class ProtoTarget(
         val outDirectory = context.outDirectory
 
         for (protoFile in schema.protoFiles) {
-          if (!context.inSourcePath(protoFile) || protoFile.isEmpty()) continue
+          if (!context.inSourcePath(protoFile) ||
+            protoFile.isEmpty() ||
+            // We never emit it.
+            protoFile.location.toString() == "google/protobuf/descriptor.proto"
```

Comment:
Do likewise for our other magic files? Like Wire extensions and stuff?

Reply:
I modified the code to never emit any runtime protos